app/src/main/java/com/sschuraytz/charcoaldrawing/DrawingView.java

```diff
@@ -26,6 +31,7 @@ public DrawingView(Context context, AttributeSet attributes) {
         super(context, attributes);
         charcoalTool = new CharcoalTool();
         eraseTool = new EraseTool();
+        voiceCommands = new VoiceCommands(this.getContext());
```

Comment:
Why do you create this inside DrawingView?

DrawingView shouldn't care about how commands work.